### PR TITLE
Fixed Typographical Error in LICENSE

### PR DESCRIPTION
**Pull request by iamrajiv**
#### Brief description of what is fixed or changed

Fixed Typographical Error in LICENSE

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by iamrajiv** [comment]
can anyone please review this @sylee957 @asmeurer

File: LICENSE [hunk 1]
```diff
@@ -29,7 +29,7 @@ DAMAGE.
 
 --------------------------------------------------------------------------------
 
-Patches taken from the Diofant project (https://github.com/diofant/diofant)
```
Comment (smichr) [inline LICENSE]: this was correct "patches [that were] taken", I believe.

File: LICENSE [hunk 2]
```diff
@@ -45,7 +45,7 @@ modification, are permitted provided that the following conditions are met:
   b. Redistributions in binary form must reproduce the above copyright
      notice, this list of conditions and the following disclaimer in the
      documentation and/or other materials provided with the distribution.
-  c. Neither the name of Diofant, or SymPy nor the names of its contributors
```
Comment (smichr) [inline LICENSE]: Niether the name Diofant, SymPy nor...

But isn't this the license that has to be included from Diofant verbatim? If so, that is not a SymPy edit to make.

**Comment by asmeurer** [comment]
@iamrajiv please do not edit the license text. It needs to stay as it is, verbatim. The explanatory text around the license can be modified. 

Also note that `multipledispatch` is the name of a [library](https://github.com/mrocklin/multipledispatch).

**Comment by asmeurer** [comment]
Except for the comma changes (which are mostly correct either way), the changes to the release README look incorrect.

**Pull request closed without merging**